.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
 *.swp
+.idea/
```

Comment:
I think its a better idea to include this as a global gitignore item for your computer then add it to the project.

Reply:
I actually don’t necessarily always avoid committing it but yeah I can understand not wanting it. I’ve reverted that change.